src/families/cosmos/app/helperV1.js

```diff
@@ -0,0 +1,76 @@
+import { CLA, errorCodeToString, INS, processErrorResponse } from "./common";
```

Comment:
We can remove the whole file and all references to helperV1 I think.

Live isn't going to support v1 of cosmos app anyway